### DISCUSSION TITLE
Which tracker currently run?

### DISCUSSION THREAD

**Discussion by xstable** [kimai/kimai#4324] [Questions & Answers] [answered]
There is a nice Widget on the dashboard, which shows how many trackers currently running.
![image](https://github.com/kimai/kimai/assets/1778068/920c0d36-20f9-4186-9602-a13541817f10)

Is there a way to see which trackers this are?
Would be very handy for controlling. 

Maybe a "mouse-over" function which show current running trackers (name with currently tracked time) would be awesome.

**Comment by kevinpapst** [comment, marked as answer]
Next release will add a link to filter them by one-click: https://github.com/kimai/kimai/pull/4321

But you can already today go to "All times" and filter for "Active records".
